static/app/components/projects/utils.tsx

```diff
@@ -0,0 +1,11 @@
+import {Organization} from 'sentry/types';
+
+/**
+ * Used to determine if viewer can see project creation button
+ */
+export function canViewerCreateProjects(organization: Organization): boolean {
```

Comment:
We've not used `Viewer` as a noun to describe the person interacting with the app. 

@evanpurkhiser any opinions on naming?

Reply:
is `User` too confusing?

as far as actor nouns go
- Customer - the org entity
- User - the individual user entity

Maybe we solidify this in our [notion glossary](https://www.notion.so/sentry/Glossary-bdc7209036d14e2799e3842b7bd590ef) 😂

Reply:
I would just call it `canCreateProject`, this is what we typically call stuff like this. The name of the parameter helps distinguish.